### ISSUE TITLE
generate complete different scaffold protein

### ISSUE THREAD

**Issue by JingWangSD** [bene837/af2seq#2]
when we try to test GPCR solubility method, after AFseq job running, the sequences all fold into helixes and Beta sheet bundles. Anything we handle not correct?  Do we need to change the initiate GPCRs structure from X-ray (any mutants or design based on nature protein) to feed into AFseq?  if you can share the example how to get soluble GPCRs form, we can test it as well.
Thanks 

here is how we run af-seq, 
from af2seq import GradientDesign,MCMCDesign
design = GradientDesign('path/to/weights','output/path')
design.design('path/to/pdb/file',
          start_seq=None ,
         iterations=500,
         lr=1e-3,
         Chains=none,
         fix_pos=None,
         recycles=0,
         clampval=0.0,
         modeltype=‘ ptm ‘ ,
  Mode = ‘ gd ‘ ,   
         MSAS= none,
        loss = loss )
mcmc = MCMCDesign('path/to/weights','output/path',random_seed=0,           
 mcmc_muts=1)
mcmc.design('path/to/pdb/file',
iterations=500)

**Comment by casperg92** [comment]
For the starting sequence generation follow the steps on the main page. This worked for us, probably you're setting the plddt loss too high, and it gets stuck in a local minimum. In the meanwhile we found that this loss works best:

loss = {"fape": 0.5, "distogram": 1.0, "ptm": 0.1, "plddt": 0.0}

Let me know if this works for you!
